src/ToDo/Business/ITaskService.cs

```diff
@@ -12,5 +12,7 @@ public interface ITaskService
 
 	ValueTask<IImmutableList<ToDoTask>> GetAsync(TaskList list, CancellationToken ct);
 
-	ValueTask<IImmutableList<ToDoTask>> GetAllAsync(string displayName = "", CancellationToken ct = default);
+	ValueTask<IImmutableList<ToDoTask>> GetAllAsync(CancellationToken ct = default);
```

Comment:
I'm wondering if this could be an extension instead.

Reply:
I'm not sure of what we're trying to achieve with the separation of getall and search. The underlying API is actually the same call to /tasks/allTasks just with different search criteria.
Perhaps ITaskService should expose GetAsync (no need for "All" as that's implied) which accepts an optional Filter object that can contain a search string and/or importance flag (this would use $filter=importance eq 'high' on the underlying REST API).

Reply:
@carldebilly / @dr1rrb do you have additional comments for this?

Reply:
Well that's because we don't want to fetch all the task when there is no search term. Even if the underlying endpoint is the same, from the application business layer, the "search" do require a non empty term.

Anyway @Robert-Louis this change has already been pushed, you will have to rebase your PR https://github.com/unoplatform/uno.todo/blob/770bb625928faae55261ac99319021e945bb0251/src/ToDo/Business/ITaskService.cs#L15